lib/perl5/FlyBase/Proforma/Allele.pm

```diff
@@ -331,7 +331,7 @@ sub process {
     }
     if (   defined( $ph{GA8} )
         && $ph{GA8} ne ''
-        && $ph{GA8} =~ /in vitro construct/ )
+        && $ph{GA8} eq 'in vitro construct' )
```

Comment:
as you said, this change isn't technically required, but I think it should do what we want once the 'in vitro construct - X' terms are obsoleted, so I am happy with this change